tests/TestCompose_nutils-of.Ubuntu1804/referenceOutput/Nutils/precice-Nutils-convergence.log

```diff
@@ -7,7 +7,7 @@ Timestep  Iteration  resNorm(0)
 1  6  0.0000014667397339  
 2  1  0.0008861429935086  
 2  2  0.0000009250774064  
-3  1  0.0005705612743593  
+3  100000  0.0005705612743593  
```

Comment:
Why this?

Reply:
This is just for the sake of provoking failure (and make really sure that #169 is really an issue)

Reply:
I will of course revert 90ff76e before merging this PR.

Reply:
Reverted. See 149a5a9.